src/Monolog/Handler/TelegramBotHandler.php

```diff
@@ -30,6 +30,11 @@
  */
 class TelegramBotHandler extends AbstractProcessingHandler
 {
+    public const MESSAGE_LENGTH_LIMIT = 4096;
+    public const TRIM_TOO_LONG_MESSAGE = 'trim';
+    public const SPLIT_TOO_LONG_MESSAGE_INTO_MULTIPLE_MESSAGES = 'split';
```

Comment:
I'd rename these to LONG_MESSAGE_TRIM and LONG_MESSAGE_SPLIT I think, then we can more easily type-hint this for phpstan/psalm as being a parameter of type `self::LONG_MESSAGE_*`.